docs/_posts/2016-09-21-this-week-in-react-1.md

```diff
@@ -0,0 +1,95 @@
+
+**Eric:** You built react-music in only two days?
+
+**Ken:** Sure did. I rewrote it fundamentally a week after it released in one day.
+
+**Eric:** Have any of your React projects brought about any interesting or unexpected uses?
+
+**Ken:** Totally. I was giving a presentation at Seattle ReactJS about Spectacle and this dude Jim Pick comes up to me and showed me how he forked spectacle and turned it into an interactive e learning presentation tool. I was amazed. Also, Simon Vrachliotis just gave a talk at React Sydney about react-music, and had this super dope UI where you could mute tracks and apply varying degrees of effects. And of course, James Kyle built a spectacle extension called spectacle-code-slide which is pretty much the best thing ever made. I always get super pumped when people do cool things and are inspired by the stuff I build.
+
+**Eric:** Are you working on anything new or exciting that you can talk about?
+
+**Ken:** Well, I suppose the next thing I build I want to be the sort of thing that people can use at work every day. I just have to think of what. Expect to see some really cool updates to the Formidable Victory project, where we are adding a tighter hyperterm integration to our victory-cli lib, and a new Victory UIExplorer app we are going to launch. I might look into making an electron or web based version of webpack dashboard, for better windows compat and because people generally want it.
+If you want to learn more about what Ken is working he will be speaking at [Seattle ReactJS on September 28, 2016](https://www.meetup.com/seattle-react-js/events/233723976/) and at [React London Meetup on March 18, 2017](https://meetup.react.london/).
+
+_Want us to interview someone from the React community? Contact [community@reactjs.org](mailto:community@reactjs.org?subject=Interview%20Request)_
+
+![A list of upcoming React community events.](/react/img/blog/thisweekinreact/upcoming.png)
+
+__Meetups__
+
+- [Thursday, September 22, 2016 San Francisco @ Minted](https://www.meetup.com/ReactJS-San-Francisco/events/231776842/)
+- [Thursday, September 22, 2016 Netherlands @ FrontMen](https://www.meetup.com/ReactJS-NL/events/233099575/)
+- [Saturday, September 24, 2016 Bangalore @ Multunus](https://www.meetup.com/ReactJS-Bangalore/events/233685933/)
+- [Wednesday, September 28, 2016 Seattle @ Redfin](https://www.meetup.com/seattle-react-js/)
+- [Thursday, September 29, 2016 Israel @ Klarna](https://www.meetup.com/ReactJS-IL/events/234262147/)
+- [Monday, October 3, 2016 Barcelona @ Lodgify](https://www.meetup.com/ReactJS-Barcelona/events/233971585/)
+- [Tuesday, October 18, 2016, London @ Facebook](https://meetup.react.london/)
+
+_Do you know about a React event coming up in the next 2 months? [Check out this event.](mailto:community@reactjs.org?subject=React%20Event)_
```

Comment:
"Check out this event" is a weird call to action. Probably want to say "Email us" or something.